examples/counter/rollup.config.js

```diff
@@ -0,0 +1,18 @@
+import node from 'rollup-plugin-node-resolve'
+import buble from 'rollup-plugin-buble'
+import flow from 'rollup-plugin-flow'
+
+export default {
+  input: 'src/index.js',
+  plugins: [
+    flow(),
+    buble(),
```

Comment:
Why buble for examples? es6 works great in the browsers and node.

Reply:
Sounds good to me.